deployment/terraform/variables.tf

```diff
@@ -301,6 +301,15 @@ variable "franklin_migrations_memory" {
   type = number
 }
 
+variable "aws_azavea_account_id" {
+  type = string
+}
+
+variable "aws_administrator_policy_arn" {
+  default = "arn:aws:iam::aws:policy/AdministratorAccess"
+  type    = string
+}
```

Comment:
We could get away with just list and put permissions for ECR, but I foresee us using this with Terraform in the future. In that case we'll need admin access.